cfgov/paying_for_college/disclosures/urls.py

```diff
@@ -1,9 +1,11 @@
 from django.urls import re_path
 
+from flags.urls import flagged_re_path
+
 from paying_for_college.views import (
     BaseTemplateView, ConstantsRepresentation, ExpenseRepresentation,
     FeedbackView, OfferView, ProgramRepresentation, SchoolRepresentation,
-    StatsRepresentation, VerifyView, school_search_api
+    StatsRepresentation, VerifyView, school_search, school_search_api
```

Comment:
```suggestion
    StatsRepresentation, VerifyView, school_autocomplete, school_search_api
```

Reply:
Done. I changed `school_search` to `school_autocomplete` in `urls.py`.